packages/storefront-sdk/__mocks__/utils/getFetchPayload.ts

```diff
@@ -1,11 +1,16 @@
 function getFetchPayload(
 	data: object,
-	responseOptions: { status: number } = {
+	responseOptions: { status: number; headers?: { [key: string]: string } } = {
```

Comment:
Doesn't change the meaning at all, but to offer up an alternative typing:

```suggestion
	responseOptions: { status: number; headers?: Record<string, string> = {
```